code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -188,7 +188,7 @@
 //secborg stun baton module
 
 /obj/item/weapon/melee/baton/robot
-	hitcost = 600
+	hitcost = 400
```

Comment:
This has no bearing on the PR. PRs should be atomic, and while this is a small change the master change is contentious enough that this change should be pushed separately. Not to mention it has nothing to do with flashes. Changes to cyborg tools should be held in a different PR.